app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -1769,6 +1788,19 @@ class VaultAddEditViewModel @Inject constructor(
 
         getRequestAndRegisterCredential()
     }
+
+    private fun handleSshKeyCipherItemsFeatureFlagReceive(
+        action: VaultAddEditAction.Internal.SshKeyCipherItemsFeatureFlagReceive,
+    ) {
+        mutableStateFlow.update {
```

Comment:
Can we add a test for this action

Reply:
That was actually a merge mistake, I've already reverted it. Thanks for catching that!